test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -371,6 +388,19 @@ void UcxContext::progress_failed_connections()
     }
 }
 
+void UcxContext::progress_disconnected_connections()
+{
+    for (std::list<UcxConnection *>::iterator it = _disconnecting_conns.begin();
+         it != _disconnecting_conns.end(); ) {
+        UcxConnection *conn = *it;
```

Comment:
can we remove "conn" local var? it's used only once

Reply:
added to improve readability:
```
        UcxConnection *conn = *it;
        if (conn->disconnect_progress()) {
...
```
vs
```
        if ((*it)->disconnect_progress()) {
...
```